Tiltfile

```diff
@@ -61,16 +62,28 @@ if local_config["helm_chart_local_repo"]["enabled"]:
 # 4. Use an init container to run the scripts for updating config from environment variables.
 # This can leverage the scripts under `e2e` package to be consistent with the CI workflow.\\
 
-# Import configuration files into Kubernetes ConfigMap
-configmap_create("path-config", from_file="local/path/config/.config.yaml", watch=True)
+local_resource(
```

Comment:
So this fixes the issue of config YAML not live reloading correctly? If so I love it.

Reply:
Yup, see the GitHub PR description. Specifically, the `How I Tested This` section to give it a shot yourself.